### PR TITLE
Tf 01 007 calendar and scheduling

### PR DESCRIPTION
**Pull request by Rasheduluctit**
Calendar and Scheduling Page done initially.
localhost_3000_pages_calendarAndscheduling
[screen-capture.webm](https://github.com/UCT-IT/edume/assets/149924448/a0a77c8d-1c56-41bf-a28e-ab1c71f40388)

### CODE REVIEW

File: src/app/pages/calendarAndscheduling/simplifiedScheduling/RightComponent.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import React from "react";
+import calendarImg5 from "../../../../../public/featuresPages/calendarAndschedulingImages/calendar-image-5.png";
+
+const RightComponent = () => {
+  return (
+    <React.Fragment>
+      <div className="w-3/4 mx-auto mt-4 md:mt-0">
+        <Image
+          src={calendarImg5}
+          alt=""
```
Comment (saminsss) [inline src/app/pages/calendarAndscheduling/simplifiedScheduling/RightComponent.tsx]: Add an alt for image

File: src/app/pages/calendarAndscheduling/stayOrganized/RightComponent.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+import React from "react";
+import calendarImg12 from "../../../../../public/featuresPages/calendarAndschedulingImages/calendar-image-12.png";
+
+const RightComponent = () => {
+  return (
+    <React.Fragment>
+      <div className="w-3/4 mx-auto mt-4 md:mt-0">
+        <Image
+          src={calendarImg12}
+          alt=""
```
Comment (saminsss) [inline src/app/pages/calendarAndscheduling/stayOrganized/RightComponent.tsx]: Add an alt

File: src/app/pages/calendarAndscheduling/uptodateRecords/RightComponent.tsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+import React from "react";
+import calendarImg10 from "../../../../../public/featuresPages/calendarAndschedulingImages/calendar-image-10.png";
+
+const RightComponent = () => {
+  return (
+    <React.Fragment>
+      <div className="w-3/4 mx-auto mt-4 md:mt-0">
+        <Image
+          src={calendarImg10}
+          alt=""
```
Comment (saminsss) [inline src/app/pages/calendarAndscheduling/uptodateRecords/RightComponent.tsx]: Add an alt

File: src/app/pages/calendarAndscheduling/websiteBookingOne/RightComponent.tsx [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+import React from "react";
+import calendarImg8 from "../../../../../public/featuresPages/calendarAndschedulingImages/calendar-image-8.png";
+
+const RightComponent = () => {
+  return (
+    <React.Fragment>
+      <div className="w-3/4 mx-auto mt-4 md:mt-0">
+        <Image
+          src={calendarImg8}
+          alt=""
```
Comment (saminsss) [inline src/app/pages/calendarAndscheduling/websiteBookingOne/RightComponent.tsx]: Add alt

File: src/app/pages/calendarAndscheduling/websiteBookingTwo/RightComponent.tsx [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+import React from "react";
+import calendarImg9 from "../../../../../public/featuresPages/calendarAndschedulingImages/calendar-image-9.png";
+
+const RightComponent = () => {
+  return (
+    <React.Fragment>
+      <div className="w-3/4 mx-auto mt-4 md:mt-0">
+        <Image
+          src={calendarImg9}
+          alt=""
```
Comment (saminsss) [inline src/app/pages/calendarAndscheduling/websiteBookingTwo/RightComponent.tsx]: Add alt

File: src/app/pages/calendarAndscheduling/zoneSupport/RightComponent.tsx [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+import React from "react";
+import calendarImg13 from "../../../../../public/featuresPages/calendarAndschedulingImages/calendar-image-13.png";
+
+const RightComponent = () => {
+  return (
+    <React.Fragment>
+      <div className="w-3/4 mx-auto mt-4 md:mt-0">
+        <Image
+          src={calendarImg13}
+          alt=""
```
Comment (saminsss) [inline src/app/pages/calendarAndscheduling/zoneSupport/RightComponent.tsx]: Add alt

**Review by saminsss — CHANGES_REQUESTED** [review]
Left a few comments. Can merge after addressing comments

File: src/app/pages/calendarAndscheduling/simplifiedScheduling/RightComponent.tsx (hunk 1 above)
Comment (Rasheduluctit) [reply src/app/pages/calendarAndscheduling/simplifiedScheduling/RightComponent.tsx]: Alt added

File: src/app/pages/calendarAndscheduling/stayOrganized/RightComponent.tsx (hunk 2 above)
Comment (Rasheduluctit) [reply src/app/pages/calendarAndscheduling/stayOrganized/RightComponent.tsx]: Added

File: src/app/pages/calendarAndscheduling/uptodateRecords/RightComponent.tsx (hunk 3 above)
Comment (Rasheduluctit) [reply src/app/pages/calendarAndscheduling/uptodateRecords/RightComponent.tsx]: Added

File: src/app/pages/calendarAndscheduling/websiteBookingOne/RightComponent.tsx (hunk 4 above)
Comment (Rasheduluctit) [reply src/app/pages/calendarAndscheduling/websiteBookingOne/RightComponent.tsx]: Added

File: src/app/pages/calendarAndscheduling/websiteBookingTwo/RightComponent.tsx (hunk 5 above)
Comment (Rasheduluctit) [reply src/app/pages/calendarAndscheduling/websiteBookingTwo/RightComponent.tsx]: Added

File: src/app/pages/calendarAndscheduling/zoneSupport/RightComponent.tsx (hunk 6 above)
Comment (Rasheduluctit) [reply src/app/pages/calendarAndscheduling/zoneSupport/RightComponent.tsx]: Added

**Pull request merged**